distributions/demo-google-deployment/bin/encrypt_and_upload_secrets.sh

```diff
@@ -81,26 +98,26 @@ else
   exit 0
 fi
 
-ENCRYPTED_SECRETS_DIR="../environments/$ENV/app_data/encrypted_secrets/"
+ENCRYPTED_SECRETS_DIR="$SECRETS_DIR/encrypted_secrets/"
 if [[ ! -e $ENCRYPTED_SECRETS_DIR ]]; then
   echo -e "Directory $ENCRYPTED_SECRETS_DIR does not already exist... creating it"
   mkdir $ENCRYPTED_SECRETS_DIR
   echo -e "Encrypting secrets..."
   encrypt_secret "FLICKR"
   encrypt_secret "GOOGLE"
-  encrypt_secret "INSTAGRAM"
-  encrypt_secret "MICROSOFT"
-  encrypt_secret "RTM"
-  encrypt_secret "SHUTTERSTOCK"
-  encrypt_secret "SMUGMUG"
-  encrypt_secret "JWT"
+  #encrypt_secret "INSTAGRAM"
```

Comment:
Why are these other ones commented out? Should we also allow a list of secrets to update as a cmd line arg? (and default to all of them if no arg is specified)

Reply:
just saw @rtannenbaum's comment, either way is fine!